pkg/scheduler/plugins/loadaware/load_aware.go

```diff
@@ -146,9 +146,29 @@ func (p *Plugin) Filter(ctx context.Context, state *framework.CycleState, pod *c
 		return nil
 	}
 
+	allocatable, err := p.estimator.EstimateNode(node)
+	if err != nil {
+		klog.ErrorS(err, "Failed to EstimateNode", "node", node.Name)
+		return framework.NewStatus(framework.Error, err.Error())
+	}
 	filterProfile := generateUsageThresholdsFilterProfile(node, p.args)
-	if len(filterProfile.ProdUsageThresholds) > 0 && extension.GetPodPriorityClassWithDefault(pod) == extension.PriorityProd {
-		status := p.filterProdUsage(node, nodeMetric, filterProfile.ProdUsageThresholds)
+	prodPod := len(filterProfile.ProdUsageThresholds) > 0 && extension.GetPodPriorityClassWithDefault(pod) == extension.PriorityProd && p.args.ScoreAccordingProdUsage
```

Comment:
Why add new condition `&& p.args.ScoreAccordingProdUsage` here?

Reply:
I made a mistake here, I thought this parameter determines whether the Prod Pod needs to be calculated separately. But it is only used for score stage. I will delete this condition